doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -324,10 +324,13 @@ PMIC samples
 
 |no_changes_yet_note|
 
-Protocol serialization samples
-------------------------------
+Protocols serialization samples
```

Comment:
protocol is correct, samples it plural, not protocols

Reply:
The samples are named "Protocols serialization client" and "Protocols serialization server" respectively so I wanted to align the changelog accordingly. The sample's names probably meant to emphasize the support for multiple protocols simultaneously. Are you saying the sample's names are not correct?

Reply:
having two plurals like that doesn't sound right in a sentence "I am using the protocols serialization samples" but I guess it fits it

Reply:
Ok, I'll revert the change if this doesn't sound right.